pkg/storage/gcs.go

```diff
@@ -5,14 +5,16 @@ import (
 	"encoding/base64"
 	"errors"
 	"fmt"
-	"google.golang.org/api/iterator"
 	"io"
 	"net/http"
 	"path"
 	"strings"
 	"time"
 
+	"google.golang.org/api/iterator"
+
 	"github.com/Altinity/clickhouse-backup/pkg/config"
+	pool "github.com/jolestar/go-commons-pool"
```

Comment:
could we rewrite with using `github.com/jolestar/go-commons-pool/v2` here?

Reply:
ok